web/src/components/AttributeRow/AttributeCheck.tsx

```diff
@@ -0,0 +1,54 @@
+import {Dropdown, Menu} from 'antd';
+
+import {useAssertionForm} from 'components/AssertionForm/AssertionFormProvider';
+import {IResult} from 'components/SpanDetail/SpanDetail';
+import * as S from './AttributeRow.styled';
+
+interface IProps {
+  items: IResult[];
+  type: 'error' | 'success';
+}
+
+const AttributeCheck = ({items, type}: IProps) => {
+  const {setIsCollapsed} = useAssertionForm();
+
+  const handleOnClick = (id: string) => {
+    setIsCollapsed(true);
+    const resultListElement = document.getElementById(`assertion-${id}`);
+    const offsetTop = resultListElement?.offsetTop ?? 0;
```

Comment:
What do you think about moving this to a hook? like useScrollTo and we can call it by passing an id 😄

Reply:
Done